### PR TITLE
Added 9-5-2024 meeting notes

### PR DESCRIPTION
**Pull request by anataliocs**
Video here:  https://drive.google.com/file/d/1bw1KCnF2C6cz0NhWNGKoUOg4oiC-OZck/view?usp=sharing

### CODE REVIEW

**Comment by anataliocs** [comment]
Verified locally:

image

**Comment by anataliocs** [comment]
Verified locally

image

File: meeting-notes/2024-09-05.mdx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+<iframe src="https://drive.google.com/file/d/1bw1KCnF2C6cz0NhWNGKoUOg4oiC-OZck/preview" width="640" height="480" allow="autoplay"></iframe>
+
+Agenda: [Discord thread](https://discord.com/channels/897514728459468821/900374272751591424/1280678171053789317)
+
+Platform team demonstrated Galexie, a part of CDP(Composable Data Platform):
+
+1. Galexie
+     1. Data Extraction: Extracts raw ledger data from the Stellar network
+     2. Compression: Compresses raw data for efficient storage
+     3. Storage Options: Supports various storage solutions, starting with Google Cloud Storage (GCS)
```
Comment (sreuland) [inline meeting-notes/2024-09-05.mdx:16]:
```suggestion
     3. Datastore: an abstraction that supports various physical storage layers, starting with Google Cloud Storage (GCS)
```

File: meeting-notes/2024-09-05.mdx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+<iframe src="https://drive.google.com/file/d/1bw1KCnF2C6cz0NhWNGKoUOg4oiC-OZck/preview" width="640" height="480" allow="autoplay"></iframe>
+
+Agenda: [Discord thread](https://discord.com/channels/897514728459468821/900374272751591424/1280678171053789317)
+
+Platform team demonstrated Galexie, a part of CDP(Composable Data Platform):
+
+1. Galexie
+     1. Data Extraction: Extracts raw ledger data from the Stellar network
```
Comment (sreuland) [inline meeting-notes/2024-09-05.mdx:14]:
```suggestion
     1. Data Extraction: Extracts raw ledger close data, the LedgerCloseMeta, from the Stellar network
```

File: meeting-notes/2024-09-05.mdx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+     2. Compression: Compresses raw data for efficient storage
+     3. Storage Options: Supports various storage solutions, starting with Google Cloud Storage (GCS)
+     4. Modes of Operation: Can operate in either batch mode or streaming mode
+2. Composable Data Platform 
+     1. Flexible Datastore:  Multiple options for physical data storage layers
```
Comment (sreuland) [inline meeting-notes/2024-09-05.mdx:19]:
```suggestion
     1. Pluggable Datastore: An abstraction that can be implemented for numerous physical data storage layers
```

File: meeting-notes/2024-09-05.mdx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+     2. Galexie:  Used to extract, compress and export data to your chosen Datastore
+     3. Transform:  Structure data in a model suitable to your application
+3. Pluggable Data Pipelines
```
Comment (sreuland) [inline meeting-notes/2024-09-05.mdx:22]:
```suggestion
```

I would just delete line 22, lines 23-24 are good indented points for Transform on line 21 instead.

**Review by sreuland — APPROVED** [review]
left a couple minor suggestions

**Review by namankumar — APPROVED** [review]
lgtm

**Pull request merged**